client/my-sites/sidebar/publish-menu.jsx

```diff
@@ -78,6 +78,18 @@ const PublishMenu = React.createClass( {
 				buttonLink: site ? '/page/' + site.slug : '/page',
 				wpAdminLink: 'edit.php?post_type=page',
 				showOnAllMySites: true,
+			},
+			{
+				name: 'media',
+				label: this.translate( 'Media' ),
+				className: 'media',
+				capability: 'upload_files',
+				queryable: true,
+				config: 'manage/media',
+				link: '/media',
+				//buttonLink: site ? '/media/' + site.slug : '/media',
+				wpAdminLink: 'upload.php',
+				showOnAllMySites: true,
```

Comment:
My mistake, you're right. This config flag was meant for the standalone page.

Unrelated comment for this line: Since the page doesn't support an all sites view, I don't think it should be included in the all sites publish bar.